wordpress-mobile/AztecEditor-iOS
# HTML Editor: Colorizing Tags

### Details:
This PR implements HTML Tag Colorization. Ref #239

### To test:
1. Launch the empty editor
2. Enter some random text + style it
3. Switch to HTML Mode
4. Verify that the HTML Tags are colorized
5. Enter some new tags by hand
6. Verify that the new tags are also colorized

Needs Review: @SergioEstevao 
Thanks in advance Sergio!

**P.s.:** Prettyfication step coming up in another PR!

cc @diegoreymendez
